pwa/src/services/auth.tsx

```diff
@@ -20,4 +23,14 @@ export const isLoggedIn = () => {
 export const logout = () => {
   setUser({});
   window.sessionStorage.removeItem("jwt");
+  window.sessionStorage.removeItem("user");
 };
+
+export const validateSession = (jwt) => {
+  const decodedJwt = jwtDecode<JwtPayload>(jwt);
+
+  if (decodedJwt?.exp >= Date.now()) {
```

Comment:
Can `decodedJwt` ever be unset (and therefore optional) here? 

If so, what _should_ happen when it's unset? I'd assume it means there's no valid `jwtToken` and the session is therefore **not validated**. 

I think currently, when the `decodedJwt` is unset, the session would be validated as valid and nothing would happen.